.gitignore

```diff
@@ -0,0 +1,2 @@
+test.md
+/.github/workflows/linters.yml
```

Comment:
- [ ] kindly replace this line with node_modules as per requirements.

Reply:
Which line am I suppose to replace?

Reply:
@Lusindiso , I am talking about Line 2 .

Reply:
So I am supposed to remove it completely and add /node_modules?